starcheck/calc_ccd_temps.py

```diff
@@ -292,22 +299,22 @@ def get_week_states(tstart, tstop, bs_cmds, tlm):
     :param tlm: available pitch and aacccdpt telemetry recarray from fetch
     :returns: numpy recarray of states
     """
-    cstates = Table(get_cmd_states.fetch_states(DateTime(tstart) - 30,
+
+    # Get temperature data at the end of available telemetry
+    ok = tlm['date'] > tlm['date'][-1] - 1400
+    init_aacccdpt = np.mean(tlm['aacccdpt'][ok])
+    init_tlm_time = np.mean(tlm['date'][ok])
+
+    # Get states at and after that time
+    cstates = Table(get_cmd_states.fetch_states(init_tlm_time,
                                                 tstop,
                                                 vals=['obsid',
                                                       'pitch',
                                                       'q1', 'q2', 'q3', 'q4']))
-    # Get the last state at least 3 days before tstart and at least one hour
-    # before the last available telemetry
-    cstate0 = cstates[(cstates['tstart'] < (DateTime(tstart) - 3).secs)
```

Comment:
I do have more temperature diffs in the regression set than I thought I would have, and I suspect most of that is due to removing this "3 days before backstop tstart" rule when running these regression weeks.  Do we need to quantify this (show the diffs are mostly at the beginning of schedules) or just run a couple of non-eclipse weeks and show they match MCC well enough?

Reply:
Run a couple of non-eclipse loads and show diffs less than 0.25 C (as you mentioned in a different thread).  There are still some state diffs from ORviewer vs. kadi or cmd_states.

Reply:
Yes.  I looked at mar1119 and mar2519 and they look good to me.

![mar1119b_diff](https://user-images.githubusercontent.com/791508/61244385-1a12bc80-a718-11e9-8ed5-bd82aa2e2a13.png)
![mar2519a_diff](https://user-images.githubusercontent.com/791508/61244386-1a12bc80-a718-11e9-85a5-d0a3644f81d2.png)

Reply:
I don't entirely understand even the small diffs, but for the ACA thermal model I was comfortable with these as at the level of "don't care".